ocs_ci/deployment/helpers/rosa_prod_cluster_helpers.py

```diff
@@ -0,0 +1,133 @@
+"""
+This module contains helpers function related to Managed Service ROSA
+Clusters in production environment
+"""
+import logging
+import os
+import re
+import time
+
+from ocs_ci.framework import config
+from ocs_ci.ocs.exceptions import ROSAProdAdminLoginFailedException
+from ocs_ci.utility.utils import exec_cmd, TimeoutSampler
+
+logger = logging.getLogger(__name__)
+
+
+class ROSAProdEnvCluster(object):
+    """
+    A base class for Managed Service ROSA Cluster in Production Environment
+    """
+
+    def __init__(self, cluster):
+        """
+        Initialize required variables
+
+        Args:
+            cluster (str): Name of the cluster in production environment
+
+        """
+        self.cluster = cluster
+        self.kubeconfig_path = os.path.join(
+            config.ENV_DATA["cluster_path"], config.RUN["kubeconfig_location"]
+        )
+        self.kubeadmin_password_path = os.path.join(
+            config.ENV_DATA["cluster_path"], config.RUN["password_location"]
+        )
+
+        # create "auth" folder
+        abs_path = os.path.expanduser(self.kubeconfig_path)
+        base_path = os.path.dirname(abs_path)
+        os.makedirs(base_path, exist_ok=True)
+
+        # login to cluster using admin account
+        self._login()
+
+    def _login(self):
+        """
+        Login to cluster
+        """
+        self.create_admin()
```

Comment:
Creation of a user should not be part of function called _login. It is also not noted in docstring.

Reply:
removed login method